Arena/src/Common/Helpers/GameNotificationHelper.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Common.Interfaces;
+
+namespace Common.Helpers
+{
+    public static class GameNotificationHelper
+    {
+        public static readonly int NotificationDelay = 2000;
+        public static string GetInitialNotification(IList<ICompetitor> competitors)
```

Comment:
What if we have more competitors? I would modify it so that:
1. IList => IEnumerable
2. string.Join (competitors, " vs ");